e2e/_suites/fleet/stand-alone.go

```diff
@@ -177,35 +142,47 @@ func (sats *StandAloneTestSuite) startAgent(image string, composeFilename string
 		return err
 	}
 
-	sats.Image = image
-	sats.Hostname = hostname
-	sats.Cleanup = true
+	fts.Image = image
+	fts.Hostname = hostname
 
-	err = sats.installTestTools(containerName)
+	err = fts.installTestTools(containerName)
 	if err != nil {
 		return err
 	}
 
 	return nil
 }
 
-func (sats *StandAloneTestSuite) getContainerLogs() error {
-	serviceManager := compose.NewServiceManager()
+func (fts *FleetTestSuite) thePolicyShowsTheDatasourceAdded(packageName string) error {
+	log.WithFields(log.Fields{
+		"policyID": fts.FleetServerPolicy.ID,
+		"package":  packageName,
+	}).Trace("Checking if the policy shows the package added")
+
+	maxTimeout := time.Minute
```

Comment:
Wdyt about using `common.TimeoutFactor` here as a multiplier (default value is 5 on CI, but configurable in Jenkins UI)? If the 1 minute is enough then I'm OK with this change.

We use the factor for CI, where sometimes we want to verify if waiting more for a condition allows the test code to reach the desired state

Reply:
Sure, it was that done way - this PR simply copies  the function, no code changes.